lambda-function.js

```diff
@@ -0,0 +1,35 @@
+module.exports = lambdaFunction;
+
+async function lambdaFunction(probot, event, context) {
+  try {
+    // Ends function immediately after callback
+    context.callbackWaitsForEmptyEventLoop = false;
+
+    // this could will be simpler once we  ship `verifyAndParse()`
+    // see https://github.com/octokit/webhooks.js/issues/379
+    await probot.webhooks.verifyAndReceive({
+      id:
+        event.headers["X-GitHub-Delivery"] ||
+        event.headers["x-github-delivery"],
+      name: event.headers["X-GitHub-Event"] || event.headers["x-github-event"],
+      signature:
+        event.headers["X-Hub-Signature-256"] ||
+        event.headers["x-hub-signature-256"] ||
+        event.headers["X-Hub-Signature"] ||
+        event.headers["x-hub-signature"],
+      payload: JSON.parse(event.body),
+    });
+
+    return {
+      statusCode: 200,
+      body: '{"ok":true}',
+    };
+  } catch (error) {
+    probot.log.error(error);
+
+    return {
+      statusCode: error.status || 500,
+      error: "ooops",
```

Comment:
Could this be more helpful? For example maybe point users at [docs for Cloudwatch logs](https://docs.aws.amazon.com/lambda/latest/dg/monitoring-cloudwatchlogs.html) on how to debug?

Reply:
I'm not sure how? By adding a "documentation_url" key to the response? I'll create a follow up issue for further discussion to unblock this PR for now